src/main/java/database/RoomDBAccess.java

```diff
@@ -19,14 +23,9 @@ public List<RoomDBModel> getRooms() {
         return getRows();
     }
 
-    @Override
-    public RoomDBModel getARoom(Integer roomID) {
-        return retrieveARow(roomID);
-    }
-
     @Override
     public void addARoom(RoomDBModel request) {
-        addARow(request.getRoomID(), request);
+        modifyARow(request.getRoomID(), request);
```

Comment:
just out of curiosity - why does `addARoom` use `modifyARow`? Aren't we making a new room instead of modifing existing room?

Reply:
modify the empty row to add a new one I think